### PR TITLE
[TTAHUB-1432] Resources associated topics widget

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

This adds a new (reusable) widget to the resource dashboard 'topics with associated resources'.

**Note:** This also contains an update for the new Trussworks pagination component.

## How to test

- Review the code.
- Make sure it shows correct data for filter, sorting, and paging.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1432

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [x] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

**Comment by GarrettEHill** [comment]
image
extra column after total?

File: frontend/src/widgets/HorizontalTableWidget.js [hunk 1]
```diff
@@ -16,39 +16,81 @@ const trimLongURLs = (url) => {
...
+      case 'desc':
+        fullAriaSort = 'descending';
+        break;
+      default:
+        fullAriaSort = 'none';
+        break;
+    }
+    return (
+      <th key={displayName.replace(' ', '_')} className={classValues || 'bg-white text-left'} scope="col" aria-sort={fullAriaSort}>
+        <a
```
Comment (thewatermethod) [inline frontend/src/widgets/HorizontalTableWidget.js:45]: please make this a ```<button>``` not an ```<a>```

File: frontend/src/components/PaginationCard.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+/* Selected Page */
+.smart-hub--pagination-card .usa-pagination .usa-current {
+  background-color: $ttahub-medium-blue;
+}
+
+/* Hover on Page Number Button */
+.smart-hub--pagination-card .usa-pagination .usa-pagination__button:hover {
+  border-color: $ttahub-medium-blue;
+  border-bottom-style: solid !important; // Prevent losing bottom border on hover.
```
[the remark this answers is not in the document]
Comment (AdamAdHocTeam) [reply frontend/src/components/PaginationCard.scss:11]: Couldn't get it to work.

File: frontend/src/widgets/HorizontalTableWidget.js (hunk 1 above)
Comment (AdamAdHocTeam) [reply frontend/src/widgets/HorizontalTableWidget.js]: This has been fixed.

File: frontend/src/widgets/HorizontalTableWidget.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 /* eslint-disable react/no-array-index-key */
+/* eslint-disable jsx-a11y/anchor-is-valid */
```
Comment (thewatermethod) [inline frontend/src/widgets/HorizontalTableWidget.js:2]: I think you can remove this now

**Pull request merged**